### DISCUSSION TITLE
Assignment Clarification

### DISCUSSION THREAD

**Discussion by metasean** [USU-ITLS-x265-23-summer/class-repo#11] [Announcements]
For assignments that require the addition of information to the class-repo, to get full credit, you need to both:

(a) submit a PR for your content (addition, modification, example, new section, etc.)

**_and_**

(b) review at least one PR from a classmate, to include merging if appropriate.

**Comment by coopsand** [comment]
Do we need to submit the PR link in canvas?

**Reply by metasean** [reply]
Nope, all PRs will be done here in GitHub.  

Most of task 3 will teach you how to make them and how to review and merge them.

Several of your classmates have already been through this, so when you get there, post your questions here, I'll let your classmates answer those questions. 😉
